### PR TITLE
Feat/add transaccion completa mall example

### PR DESCRIPTION
**Pull request by eugenioLopezRamos**
Add transaccion completa mall

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    from transaccion_completa import bp as transaccion_completa_bp
+    from patpass_by_webpay import bp as patpass_by_webpay_bp
+    from transaccion_completa_mall import bp as transaccion_completa_mall_bp
+
+    app.register_blueprint(transaccion_completa_bp, url_prefix='/fulltransaction')
+    app.register_blueprint(patpass_by_webpay_bp, url_prefix='/patpass-webpay')
+    app.register_blueprint(transaccion_completa_mall_bp, url_prefix='/mallfulltransaction/')
+
+    @app.route('/')
+    def hello_world():
```
Comment (iescaida) [inline __init__.py]: change hello_world for a more appropriate name :)

**Review by iescaida — CHANGES_REQUESTED** [review]
good work, I'm only requesting a name change, left a comment, cheers!

File: __init__.py (hunk 1 above)
Comment (eugenioLopezRamos) [reply __init__.py]: Fixed

**Review by iescaida — APPROVED** [review]
it looks good mate

**Pull request merged**